SamMou.Api.Graph/QueryObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using GraphQL;
+using GraphQL.Types;
+using SamMou.Api.Models;
+using SamMou.Api.Services.Interface;
+
+namespace SamMou.Api.GraphQL
+{
+    public class QueryObject : ObjectGraphType<object>
```

Comment:
Or, is the better practice is the class itself to be name as weatherForecastObject.
should we create another namespace for schema specifically?

Eg: https://github.com/graphql-dotnet/example-orders/blob/master/Orders/Schema/OrdersQuery.cs

So, SamMou.Api.GraphQL.Schema